app/serializers/user.js

```diff
@@ -5,9 +5,13 @@ export default ApplicationSerializer.extend({
   serialize(snapshot, options) {
     const json = this._super(...arguments);
     if (snapshot.id) {
-      let attributesToOmit = ['email'];
+      let attributesToOmit = [];
       if (!snapshot.adapterOptions || !snapshot.adapterOptions.includePassword) {
         attributesToOmit.push('password');
+      } else if (!snapshot.adapterOptions || !snapshot.adapterOptions.includeEmail) {
```

Comment:
@niranjan94 Pushed the code. I am using the save method from the beginning itself and following the same as the profile page. Please guide if I am wrong. Also I tried logging at this place:
```
else if (options && options.includeId) {
      console.log('post');
      json.data.attributes = pick(json.data.attributes, ['email', 'password']);
    }
```
but 'post' is not getting logged. So I guess it should be patch.